HappyCamps/src/app/services/auth.service.ts

```diff
@@ -0,0 +1,37 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+  login_url:string ='https://localhost:7027/api/User/login'
+  
+  constructor(private http:HttpClient, private router:Router) {}
+
+  signUp(){
+
+   }
+
+  login(loginObj:any){
+      return this.http.post<any>(this.login_url,loginObj);
```

Comment:
do we really want to accept the loginObj as any? cant we use something more specific to prevent a "wrongfully use" of this method?

Reply:
We need to create the models for this project.

Reply:
Please create it :)

Reply:
Done.